test/martha/_helpers.js

```diff
@@ -0,0 +1,58 @@
+const sinon = require('sinon');
+
+const {
+    MARTHA_V3_ALL_FIELDS,
+} = require("../../martha/martha_fields");
+
+
+const mockRequest = (req, options={}, auth="Bearer XYZ_TerraAuthToken") => {
```

Comment:
the original implementation of this helper function [is placed inside of the main testing code](https://github.com/broadinstitute/martha/blob/dev/test/martha/martha_v3.test.js#L68), and relies on global variables defined there.

Helpers should be small, pure (in the functional sense), and be separated in another file/namespace. (Namespaces help with cognitive overhead)